src/app/Http/Controllers/ConnectController.php

```diff
@@ -35,21 +35,35 @@ public function __construct(UserRepository $repository, AuthValidatorInterface $
     public function connect()
     {
         try {
-            $user = $this->repository->findByField('username', 'admin@vmms.vn')->first();
-            
+            $token = JWTAuth::getToken();
+            if (empty($token)) {
+                return response([
+                    'message' => 'The Authorization data was invalid.',
+                    "errors" => 'Bearer Token was not found',
+                ]);
+            }
+            $email = $this->JWTDecode($token);
+            $data = [
+                'username' => explode('@', $email)[ 0 ],
+                'email' => $email,
+            ];
+            $user = $this->repository->findByField('email', $email)->first();
             if (!$user) {
-                $user = $this->repository->findByField('email', 'admin@vmms.vn')->first();
-                if (!$user) {
-                    throw new NotFoundException('user');
-                }
+                $this->repository->create($data);
+            } else {
+                $token = JWTAuth::fromUser($user);
             }
-
-            $token = JWTAuth::fromUser($user);
-
         } catch (JWTException $e) {
             return response()->json(['error' => 'could_not_create_token'], 500);
         }
-
         return $this->response->array(compact('token'));
     }
+
+    public function JWTDecode($token)
```

Comment:
@minhquang19 Chỗ này em không dùng dc hàm của JWT có lẽ vì thiếu biến `JWT_SECRET ` trong file .env.
Em thử làm theo hướng dẫn này để thêm biết đó trong env và thử dùng lại hàm của JWT xem có dc ko.
https://github.com/webpress-vn/core#config-and-migration